src/Extension/FormTypeExtension.php

```diff
@@ -16,27 +16,63 @@
 use Twig\Extension\AbstractExtension;
 use Twig\Extension\GlobalsInterface;
 
-final class FormTypeExtension extends AbstractExtension implements GlobalsInterface
-{
-    /**
-     * @var bool
-     */
-    private $wrapFieldsWithAddons;
-
-    public function __construct($formType)
+if (class_exists('Twig_Extension_GlobalsInterface')) {
+    class FormTypeExtension extends AbstractExtension implements GlobalsInterface
     {
-        $this->wrapFieldsWithAddons = (true === $formType || 'standard' === $formType);
-    }
+        /**
+         * @var bool
+         */
+        private $wrapFieldsWithAddons;
 
-    public function getGlobals(): array
-    {
-        return [
-            'wrap_fields_with_addons' => $this->wrapFieldsWithAddons,
-        ];
-    }
+        public function __construct($formType)
```

Comment:
Could we add type hint for `$formType`?

Reply:
We can't. $formType can be string or bool

Reply:
I'm not asking for a type declaration, but for a type hint (`@param string|bool $formType`).

Reply:
Little confusing, I think type hint is:
```php
public function __construct(string $formType)
```
and phpdoc is:
```php
/**
* @param string $formType
*/
public function __construct($formType)

Anyway write phpdoc to deprecated code is not necessery.